All_home_works/hw9/task3.py

```diff
@@ -0,0 +1,45 @@
+"""
+Write a function that takes directory path, a file extension and an optional tokenizer.
+It will count lines in all files with that extension if there are no tokenizer.
+If a the tokenizer is not none, it will count tokens.
+For dir with two files from hw1.py:
+universal_file_counter(test_dir, "txt")
+6
+universal_file_counter(test_dir, "txt", str.split)
+6
+"""
+import os
+from pathlib import Path
+from typing import Callable, Optional
+
+from _pytest import pathlib
+
+
+def universal_file_counter(
+    dir_path: Path, file_extension: str, tokenizer: Optional[Callable] = None
+) -> int:
+    counter = 0
+    count_function = count_lines if tokenizer is None else count_tokens
+    for file_path in get_files_from_directory(dir_path, file_extension):
+        with open(file_path) as file:
+            counter += count_function(file, tokenizer)
```

Comment:
Is it better to use `count_lines(file, tokenizer)` instead of `(*args, `**kwargs)` because it's more obvious?